modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesMockAdapter.java

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
Can we get rid of the mock adapter for story?

Reply:
Let's keep it for now?